src/TrendsOverview.svelte

```diff
@@ -380,7 +379,11 @@
     <!-- Map attribution line -->
     <div class="map-attribution">
       <p class="map-attribution-text">
-        Chart includes geographic data from the US Census Bureau
+        {#if selectedCountryName == "United States"}
```

Comment:
We might want to move this content to the metadata so that we just need make a single binding to country_data[“map_source”]